database/sqlite/sqlite_health.c

```diff
@@ -362,46 +378,67 @@ void sql_health_alarm_log_save(RRDHOST *host, ALARM_ENTRY *ae)
 /* Health related SQL queries
    Get a count of rows from health log table
 */
-#define SQL_COUNT_HEALTH_LOG_DETAIL "SELECT count(1) FROM health_log_detail hld, health_log hl where hl.host_id = @host_id and hl.health_log_id = hld.health_log_id;"
-void sql_health_alarm_log_count(RRDHOST *host) {
+#define SQL_COUNT_HEALTH_LOG_DETAIL "SELECT count(1) FROM health_log_detail hld, health_log hl " \
```

Comment:
I did not get the necessity to use the function `count`, why are we not using only `SELECT 1 FROM ... `? :thinking:

Reply:
Need the number of entries as in select count(*), not just 1